packages/react/src/themes/teams/components/Popup/popupContentStyles.ts

```diff
@@ -9,6 +9,7 @@ const popupContentStyles: ComponentSlotStylesInput<PopupContentProps, PopupConte
 
     return {
       display: 'block',
+      textAlign: 'left',
```

Comment:
as we are expecting this to be applied to _any_ popup content (i.e. the one that is provided not as an object prop, but as regular react element), like in the following case:

```jsx
<Popup ... content={<div>...</div>}>
```

then we need to apply this to `Popup`'s `popup` slot in general